test/query.test.js

```diff
@@ -1906,17 +1906,19 @@ describe('query', () => {
   test('and', () => {
     var p1 = assertQuery(query.And(true, true, false), false)
     var p2 = assertQuery(query.And(true, true, true), true)
-    var p3 = assertQuery(query.And(true), true)
-    var p4 = assertQuery(query.And(false), false)
-    return Promise.all([p1, p2, p3, p4])
+    // var p3 = assertQuery(query.And(true), true)
+    // var p4 = assertQuery(query.And(false), false)
+    // return Promise.all([p1, p2, p3, p4])
```

Comment:
What's the commented code?

Reply:
@nathaniel-may I uncommented these. Originally, I needed to comment them out because we had removed `And` and `Or`'s ability to take a single argument. @marrony added that back to Core, so now the tests will pass without them being commented out.